packages/now-node/test/helpers.test.js

```diff
@@ -307,6 +308,14 @@ describe('res.status', () => {
   });
 });
 
+describe('res.redirect', () => {
+  test('res.redirect() should redirect without statusCode', () => {
+    mockListener.mockImplementation((req, res) => {
+      res.redirect('example.com');
+    });
```

Comment:
This test is not testing the behaviour of `res.redirect`. Could you add tests to check that the header and status code is correctly added in the response?

Reply:
You can use something along these lines:
```
      const res = await fetchWithProxyReq(url);
      expect(res.status).toBe(...);
      expect(res.headers.location).toBe(...);
```